apollo-openapi/src/main/java/com/ctrip/framework/apollo/openapi/client/exception/ApolloOpenApiException.java

```diff
@@ -17,8 +17,9 @@
 package com.ctrip.framework.apollo.openapi.client.exception;
 
 public class ApolloOpenApiException extends RuntimeException {
+  private static final long serialVersionUID = 6086899365233189580L;
```

Comment:
Why do we need this for an exception class?

Reply:
Fixed, thx